SpatialGDK/Documentation/content/cloud-deployment-workflow.md

```diff
@@ -75,11 +75,11 @@ You can launch a cloud deployment using the Unreal Editor or the SpatialOS CLI.
     Build out the simulated player clients (which will run on Linux in the cloud) using the following command:</br></br>
     Engine plugin filepath (default):</br>
     ```
-    UnrealEngine\Engine\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat <YourProject>SimulatedPlayer Linux Development <YourProject>.uproject 
+    UnrealEngine\Engine\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat <YourProject>SimulatedPlayer Linux Development <YourProject>.uproject -skipshadercompile
```

Comment:
The code behind -skipshadercompile isn't actually public yet. It was committed to 4.22-SpatialOSUnrealGDK-preview so I need to wait for confirmation for when it will be publicly released. We may have to pull this change from the PR for now.

We should also add this flag in a separate comment as an option, rather than always suggesting it, because I'm not sure that every game will need it. There should also be an explanation about why you might want it. Something like:
"By default, an Unreal Linux client build will attempt to re-build all shaders for the OpenGL Shading Language. You may find that when building out your Linux clients, shaders take a long time to build and fail often. It's not necessary to have them built for linux clients, so you can skip shader compilation by adding the flag `-skipshadercompile` to the above command."

Reply:
Hmm I think this is just a UAT flag that `Buildworker.bat` passes on so it should work on release branch also. there hasn't been a change to that file from 0.6.2 (which `release` is on) and 0.7.0 (which `preview` is on)

I will double check

Reply:
and yes if we can keep it in, will add the explanation

Reply:
the -skipshadercompile flag has been removed from this PR and added to this one (which targets docs-preview, and has all the same changes plus an expandable callout on -skipshadercompile) 
https://github.com/spatialos/UnrealGDK/pull/1500